packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -184,6 +184,11 @@ export class SDKClient {
     }
 
     async submitEthereumTransaction(transactionBuffer: Uint8Array): Promise<TransactionResponse> {
+        if (transactionBuffer.length > 4096) {
```

Comment:
Q: doesn't the sdk make this check for you?
If it's possible to just call the `EthereumFlow` always let's do that